robotiq_arg85_description:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
# finger right--------------------------------------
  r_finger_joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_finger_joint
    pid: {p: 1000.0, i: 0.01, d: 5}
  r_finger_joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_left_inner_knuckle_joint
    pid: {p: 1000.0, i: 0.01, d: 5}
  r_finger_joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_left_inner_finger_joint
    pid: {p: 1000.0, i: 0.01, d: 5}
  r_finger_joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_right_outer_knuckle_joint
    pid: {p: 1000.0, i: 0.01, d: 5}
  r_finger_joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_right_inner_knuckle_joint
    pid: {p: 1000.0, i: 0.01, d: 5}
  r_finger_joint6_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_right_inner_finger_joint
    pid: {p: 1000.0, i: 0.01, d: 5}

# finger left--------------------------------------
  l_finger_joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_finger_joint
    pid: {p: 1000.0, i: 0.01, d: 5}
  l_finger_joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_left_inner_knuckle_joint
    pid: {p: 1000.0, i: 0.01, d: 5}
  l_finger_joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_left_inner_finger_joint
    pid: {p: 1000.0, i: 0.01, d: 5}
  l_finger_joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_right_outer_knuckle_joint
    pid: {p: 1000.0, i: 0.01, d: 5}
  l_finger_joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_right_inner_knuckle_joint
    pid: {p: 1000.0, i: 0.01, d: 5}
  l_finger_joint6_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_right_inner_finger_joint
    pid: {p: 1000.0, i: 0.01, d: 5}